src/Surfnet/StepupRa/RaBundle/Controller/RaLocationController.php

```diff
@@ -56,21 +56,21 @@ public function manageAction(Request $request)
         $this->get('logger')->notice('Starting search for locations');
 
         $profile = $this->getProfileService()->findByIdentityId($identity->id);
-        $choices = $profile->getRaaInstitutions();
-        $institution = reset($choices);
+
+        if ($this->isGranted('ROLE_SRAA')) {
+            $institution = $identity->institution;
+            $choices = $this->getInstitutionListingService()->getAll();
+        } else {
+            $choices = $profile->getRaaInstitutions();
```

Comment:
There is no `getRaaInstitutions` on the Profile DTO, resulting in a fatal error.

Reply:
I've updated the MW client bundle